core/templates/base-components/oppia-root.directive.ts

```diff
@@ -21,33 +21,35 @@
 import { OppiaAngularRootComponent } from
   'components/oppia-angular-root.component';
 
-angular.module('oppia').directive('oppiaRoot', ['$translate',
-  function($translate) {
-    return {
-      template: require('./oppia-root.directive.html'),
-      scope: {},
-      transclude: true,
-      controllerAs: '$ctrl',
-      controller: ['$scope',
-        function($scope) {
-          $scope.initialized = false;
+angular.module('oppia').directive(
```

Comment:
I'm sorry but I'm personally not liking the line break here. I've seen few examples online and the directive name is mentioned in the same line itself. Same for all other examples on directives/controllers.

What do you say @srijanreddy98 @DubeySandeep ?

Reply:
Also, @Hudda I see not all examples are having break after `.directive(`.